zerver/openapi/zulip.yaml

```diff
@@ -9018,67 +9026,78 @@ paths:
                     Whether the client is sending a ping-only request, meaning it only
-                    wants to update the user's presence status.
+                    wants to update the user's presence `status` on the server.
 
-                    Otherwise, also requests the server to return presence data of all
-                    users in the realm, further affected by the `last_update_id`
-                    parameter.
+                    Otherwise, also requests the server return user presence data for all
+                    users in the organization, which is further specified by the
+                    [`last_update_id`](#parameter-last_update_id) parameter.
                   example: false
                   default: false
                 slim_presence:
+                  type: boolean
                   description: |
-                    Legacy parameter for configuring the format in which the server will
-                    return presence data for the realm. Modern clients should pass
-                    `last_update_id`, which implies the client expects the modern format,
-                    and should not pass this property unless interacting with an older
-                    server.
+                    Legacy parameter for configuring the format (modern or legacy) in
+                    which the server will return user presence data for the organization.
+
+                    Modern clients should use
+                    [`last_update_id`](#parameter-last_update_id), which guarantees that
+                    user presence data will be returned in the modern format, and
+                    should not pass this parameter as `true` unless interacting with an
+                    older server.
 
                     Legacy clients that do not yet support `last_update_id` may use the
-                    value of `true` to request the modern format. The legacy format, sent
-                    when `slim_presence` is `false`, will be removed entirely in a future
-                    release.
+                    value of `true` to request the modern format for user presence data.
 
-                    **Changes**: **Deprecated** in Zulip 9.0 (feature level 263);
-                    using the modern `last_update_id` API is the recommended way to
-                    request the modern format.
+                    **Note**: The legacy format for user presence data will be removed
+                    entirely in a future release.
 
-                    New in Zulip 3.0 (Unstable with no feature level yet).
-                  type: boolean
+                    **Changes**: **Deprecated** in Zulip 9.0 (feature level 263). Using
+                    the modern `last_update_id` parameter is the recommended way to
+                    request the modern format for user presence data.
+
+                    New in Zulip 3.0 (no feature level as it was an unstable API at that
+                    point).
```

Comment:
I wasn't consistent with how these are noted regarding the lack of feature level.

Also, I feel like there should be a note about at what point the modern format was both the user ID and the two timestamps.

Reply:
I think the modern format was introduced long enough ago that there's not a lot of value in having it written down, but yeah we probably should figure it out.